libs/experimental/langchain_experimental/data_anonymizer/faker_presidio_mapping.py

```diff
@@ -27,8 +27,8 @@ def get_pseudoanonymizer_mapping(seed: Optional[int] = None) -> Dict[str, Callab
             fake.random_choices(string.ascii_lowercase + string.digits, length=26)
         ),
         "IP_ADDRESS": lambda _: fake.ipv4_public(),
-        "LOCATION": lambda _: fake.address(),
-        "DATE_TIME": lambda _: fake.iso8601(),
+        "LOCATION": lambda _: fake.city(),
```

Comment:
nitpick but it is a sneaky way to have this changes ;) it is not related to multilngual anonymization

Reply:
I would leave it here, as it is comehow connected and it's sometimes difficult and time-consuming to get your PR merged by authors